DISCOVER/08_grants_scholarships.md

```diff
@@ -36,7 +37,7 @@ Registration vs. Travel &amp; Lodging
 - How to attract funding for scholarships?
   - Consider posting an opportunity for attendees or community members to make individual donations in support of travel scholarships.
   - Make corporate sponsor levels specifically tied to travel+lodging and/or registration scholarships. (e.g. [PyData offers Diversity Scholarships as sponsor benefits](https://pydata.org/pdf/sponsor-prospectus.pdf))
-  - Run a crowdfunding campaign to support particular inclusion efforts (e.g. childcare expenses — Open Source Bridge [did this](http://opensourcebridge.org/blog/2016/05/childcare-and-scholarships/).)
+  - Run a crowdfunding campaign to support particular inclusion efforts <!--(e.g. childcare expenses — Open Source Bridge ~~[did this](http://opensourcebridge.org/blog/2016/05/childcare-and-scholarships/).)-->
```

Comment:
available at: https://web.archive.org/web/20160527052000/http://opensourcebridge.org/blog/2016/05/childcare-and-scholarships/